tests/cephfs/CEPH-11219.py

```diff
@@ -280,7 +283,7 @@ def run(ceph_cluster, **kw):
                     dirs[0],
                     0,
                     1,
-                    iotype="crefi",
+                    iotype="smallfile",
                 )
             print("-------------------------------------------------------")
```

Comment:
can you also remove this `print`

Reply:
Hi Rakesh,
I removed all the print statements.